### PR TITLE
Deprecate the dbus-glib dependency

### PR DESCRIPTION
**Pull request by smallorange**
Since the dbus-glib was already deprecated, the related portion of codes for lpmd can be removed. This work entirely deprecated dbus-glib and made the dbus implementation based on GDbus.

### CODE REVIEW

**Comment by smallorange** [comment]
Okay, I'll rebase it.

File: configure.ac
Comment (yselkowitz) [inline configure.ac:60]: Why is glib 2.80 required?

Comment (smallorange) [reply configure.ac:60]: Thank you for reviewing it. 

I think glib-2.0 is enough. I'll push a new version. :)

Comment (yselkowitz) [reply configure.ac]: Technically glib and gobject don't need to be explicit, as gio and gmodule automatically include those by definition.  But ultimately that is a style question that will be up to the maintainers.

**Review by yselkowitz — APPROVED** [review]
LGTM, builds successfully.

/cc @zhang-rui

**Comment by smallorange** [comment]
Thank you :)

**Comment by zhang-rui** [comment]
can you rebase on latest main branch?
I have to revert a pull request first because that causes a regression.

File: configure.ac
Comment (smallorange) [reply configure.ac]: "glib-2.0  gobject-2.0" are for compiling tools/intel_lpmd_control.c.

**Comment by smallorange** [comment]
Okay.

**Comment by smallorange** [comment]
Hi
 
I updated this PR and I'll also update the RHEL package to the latest commit after this work is merged.

Thank you :)

**Pull request merged**
